charts/karpenter/README.md

```diff
@@ -45,7 +45,10 @@ helm upgrade --install --namespace karpenter --create-namespace \
 | controller.metrics.port | int | `8000` | The container port to use for metrics. |
 | controller.outputPaths | list | `["stdout"]` | Controller outputPaths - default to stdout only |
 | controller.resources | object | `{}` | Resources for the controller pod. |
-| controller.securityContext | object | `{}` | SecurityContext for the controller container. |
+| controller.securityContext | object | `{"allowPrivilegeEscalation":false,"capabilities":{"drop":["ALL"]},"readOnlyRootFilesystem":true}` | SecurityContext for the container. |
```

Comment:
I think we could just hardcode the security related parameters since I don't see a use-case for a user changing these. wdyt?

Reply:
@bhargaje I think it probably makes sense to hardcode this since I can't think of a good reason a user would want to make the karpenter containers less secure.

Reply:
Pushed this down into static values in the deployment